README.md

```diff
@@ -1237,6 +1237,48 @@ Other Style Guides
 
   - [15.4](#15.4) <a name='15.4'></a> For more information see [Truth Equality and JavaScript](http://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/#more-2108) by Angus Croll.
 
+  - [15.5](#15.5) <a name='16.1'></a> Avoid lexical declarations (e.g. `let`, `const`, `function`, and `class`) in `case` and `default` clauses.
```

Comment:
switch statements are relevant to comparison - my thinking was that Section 15 is the right place for constructs used for equality or comparison :-)